src/DIRAC/ConfigurationSystem/Agent/Bdii2CSAgent.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, *args, **kwargs):
         self.host = "cclcgtopbdii01.in2p3.fr:2170"
         self.glue2URLs = []
         self.glue2Only = True
-
+        self.injectOneCore = False
```

Comment:
Minor, but I would prefer this to be
```suggestion
        self.injectSingleCoreQueues = False
```
to be consistent with he option name.

Reply:
Changed the variable name